src/App.jsx

```diff
@@ -23,16 +62,46 @@ function App(props) {
     </div>
   ));
 
+  //   // Deal button text changes at end of game to start again
+  const ButtonText = cardDeck.length === 0 ? "Deal Again" : "Reset";
+
+  const handleReset = () => {
+    setCardDeck(makeShuffledDeck);
+    setp1RoundsWon(0);
+    setp2RoundsWon(0);
+    setCurrCards([]);
+    setgameStart(false);
+  };
+
   return (
     <>
       <div>
         <img src={logo} className="logo" alt="Rocket logo" />
       </div>
       <div className="card">
         <h2>React High Card 🚀</h2>
-        {currCardElems}
+        <h3>No. of cards left: {cardDeck.length}</h3>
+        <h3>No. of rounds left: {numRoundsLeft}</h3>
+        <h4>
+          <Row className="table-row">
+            <Col className="table-col">
+              <p> Player 1 drew :{currCardElems[0]} </p>
```

Comment:
What if the player hasn't drawn anything yet? This will be empty, right? Maybe we could use conditional rendering to make it a bit nicer